docs/baking.md

```diff
@@ -105,125 +101,141 @@ In order to import the snapshot, run the following command:
-`/etc/default/tezos-baker-<proto>` and `/etc/default/tezos-endorser-<proto>`.
-By default, both these daemons have `/var/lib/tezos/.tezos-client` set as a `DATA_DIR`.
-
-Additionally, you need to specify `NODE_DATA_DIR` in the `/etc/default/tezos-baker-<proto>`
-to point at the desired node data directory, e.g. `/var/lib/tezos/node-<network>`.
-
+By default `tezos-baking-<network>.service` will use the `baker` alias for the
+key that will be used for baking and endorsing.
 
+<a name="import"></a>
 ### Importing the baker key
 
 Import your baker secret key to the data directory. There are multiple ways to import
 the key:
 
-1) You have faucet JSON file from https://faucet.tzalpha.net/.
+1) The secret key is stored on a ledger.
 
-In order to activate account run:
+Open the Tezos Wallet app on your ledger and run the following
+to import the key:
+```
+sudo -u tezos tezos-client import secret key baker <ledger-url>
+```
+Apart from importing the key, you'll also need to set it up for baking. Open Tezos Baking app
+on your ledger and run the following:
 ```
-sudo -u tezos tezos-client activate account <alias> <path-to-downloaded-json>
+sudo -u tezos tezos-client setup ledger to bake for baker
 ```
 
 2) You know either the unencrypted or password-encrypted secret key for your address.
 
 In order to import such a key, run:
 ```
-sudo -u tezos tezos-client import secret key <alias> <secret-key>
+sudo -u tezos tezos-client import secret key baker <secret-key>
 ```
-3) The secret key is stored on a ledger.
 
-Open the Tezoz Wallet app on your ledger and run the following
-to import the key:
-```
-sudo -u tezos tezos-client import secret key <alias> <ledger-url>
-```
-Apart from importing the key, you'll also need to set it up for baking. Open Tezos Baking app
-on your ledger and run the following:
+3) You have faucet JSON file from https://faucet.tzalpha.net/.
+
+In order to activate account run:
 ```
-sudo -u tezos tezos-client setup ledger to bake for <alias>
+sudo -u tezos tezos-client activate account baker with <path-to-downloaded-json>
 ```
 
+<a name="registration"></a>
 ### Registering the baker
 Once the key is imported, you'll need to register your baker, in order to do that run the following
 command:
 ```
-sudo -u tezos tezos-client register key <alias> as delegate
+sudo -u tezos tezos-client register key baker as delegate
 ```
 
 Check a blockchain explorer (e.g. https://tzkt.io/ or https://tzstats.com/) to see the baker status and
 baking rights of your account.
 
-### Updating baker and endorser daemons configuration
-Apart from that, you'll need to update the `BAKER_ACCOUNT` and `ENDORSER_ACCOUNT` (in
-`/etc/default/tezos-baker-<proto>` and `/etc/default/tezos-endorser-<proto>` respectively) in
-accordance to the **alias** of the imported key.
+## Starting baking instance
 
-### Starting daemons
+Once the key is imported and the configs are updated (if needed), you can start your baking instance:
```

Comment:
There is no previous mention of config update before this anymore, so this is quite confusing to read, I suggest:
```suggestion
Once the key is imported and the baker registered, you can start your baking instance:
```